commit 06a8d6478752403a617ed9fb9f6b78c31ad25020
Author: Joel A Fernandes <agnel.joel@gmail.com>
Date:   Tue Jun 21 17:31:13 2011 -0500

    omap: Disable UART sleep timeout to avoid serial port problems on wakeup.
    
    Signed-off-by: Joel A Fernandes <agnel.joel@gmail.com>

diff --git a/arch/arm/mach-omap2/serial.c b/arch/arm/mach-omap2/serial.c
index 19805a7..35a24af 100644
--- a/arch/arm/mach-omap2/serial.c
+++ b/arch/arm/mach-omap2/serial.c
@@ -36,7 +36,7 @@
 #define UART_OMAP_NO_EMPTY_FIFO_READ_IP_REV	0x52
 #define UART_OMAP_WER		0x17	/* Wake-up enable register */
 
-#define DEFAULT_TIMEOUT (5 * HZ)
+#define DEFAULT_TIMEOUT 0
 
 struct omap_uart_state {
 	int num;
